### PR TITLE
Implement ability to like a song

### PR DESCRIPTION
**Pull request by Slawisch**

### CODE REVIEW

File: backend/Perflow/Services/SongReactionService.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                throw new ArgumentException(ex.Message);
+            }
+            catch (Exception ex)
+            {
+                throw new Exception(ex.Message);
+            }
+        }
+
+    }
+}
```
Comment (NikitaKrasnovSW) [inline backend/Perflow/Services/SongReactionService.cs]: Rrefactor to separate methods. Like song should not be responsible for dislike

File: backend/Perflow/Services/SongReactionService.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+                if (likes.Any())
+                {
+                    _context.SongReactions.RemoveRange(likes);
+                    await _context.SaveChangesAsync();
+
+                    return false;
+                }
+
+                var songReaction = new SongReaction(){SongId = songReactionDto.SongId, UserId = songReactionDto.UserId};
```
Comment (NikitaKrasnovSW) [inline backend/Perflow/Services/SongReactionService.cs]: Fix formatting

File: frontend/src/app/components/shared/song-toolbar/song-toolbar.component.html [hunk 3]
```diff
@@ -16,7 +16,11 @@ <h6 class="songname">{{songForPlay.name}}</h6>
             </div>
         </div>
         <div class="midpart">
+<<<<<<< HEAD
```
Comment (NikitaKrasnovSW) [inline frontend/src/app/components/shared/song-toolbar/song-toolbar.component.html]: The file is invalid

File: frontend/src/app/services/reaction.service.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import { HttpInternalService } from './http-internal.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ReactionService {
+  httpService: HttpInternalService;
+  constructor(httpService: HttpInternalService) {
+    this.httpService = httpService;
+  }
```
Comment (NikitaKrasnovSW) [inline frontend/src/app/services/reaction.service.ts]: Add new lines

**Review by NikitaKrasnovSW — CHANGES_REQUESTED** [review]

File: backend/Perflow/Services/SongReactionService.cs (hunk 1 above)
Comment (Slawisch) [reply backend/Perflow/Services/SongReactionService.cs]: Should I create 2 separated endpoints for it? Or it'll be enough to separate methods in service?

**Review by NikitaKrasnovSW — APPROVED** [review]

**Pull request merged**